libs/frontend/feature/association/role-page/src/lib/role-page/role-page.formly.ts

```diff
@@ -11,4 +11,12 @@ export const createRoleFormly: FormlyFieldConfig[] = [
       required: true,
     },
   },
+  {
+    key: 'associationId',
+    type: Form.NumericInput,
+    templateOptions: {
+      label: "Id de l'association",
+      required: true,
+    },
+  },
```

Comment:
Peux tu mettre un TODO: To be automated ?
Car on doit connaitre l'id et non l'utilisateur